C-Plus-Plus/graphs/subtree_size.cpp

```diff
@@ -0,0 +1,64 @@
+/*Given the number of nodes and edges of graph print an array containg subtree size of each node*/
+
+#include <iostream>
+#include <vector>
+using namespace std;
+
+vector<int> arr[50];        //Adjacency List storing the graph
+vector<int> vis(50, 0);     //Visited array, it stores wheather the node is visited or not
+vector<int> subsize(50, 0); //Subsize array, it stores the subtree size of each node
```

Comment:
You need to add comments to your code. Make sure the comments you add are above the line of code and not alongside it.